src/main/webapp/features.jsp

```diff
@@ -2,187 +2,191 @@
 <t:staticPage currentPage="features">
     <div class="container">
         <div class="lead">
-            Here is an overview of some notable TEAMMATES features.
-            The <a href="instructorHelp.jsp">help page</a> contains more details about how to use these features.
+            <h1 class="color_orange">
+                Features
+            </h1>
+            <p style="font-size:14px"> Here is an overview of some notable TEAMMATES features.
```

Comment:
Please edit the CSS `src/main/webapp/stylesheets/teammatesCommon.css` instead of using inline styles. Applies to the rest of the inline styles throughout. By editing the CSS, the changes would propagate to the rest of the pages and make them consistent.